### ISSUE TITLE
Log in screen functional issues

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#7821]
**Issues**

- [x] Reset Password button should be dimmed until text is entered
- [x] Reset with invalid email format just get stuck spinning
- [ ] After creating account and clicking link from email.  After entering password information, page spins with "Loading" message but never loads.
- [x] Clicking on resend link from email after already resetting, gives me a 500 page.
- [x] Reset Password just spins if I enter nothing or a username that doesn't exist.  Should check and if username doesn't exist, respond with error message.
- [x] Click on language drop-down, there is a blank field that is un-clickable in list
- [ ] If I sit on the log in screen, it eventually gives me a "Your session has timed out.  Log in again to continue".  

**Optional Issues:**
- [ ] Why have Name field for create new?
- [x] In account already exist, have link to go to reset password

**Comment by westlywright** [comment]
@tfiduccia I've left a few of the issues unresolved

RE: Clicking link from email and just loading, I was not able to repro this but I fixed a BUNCH of error messages so it should error out properly on the server side and if you see this now it should show an error. 
RE: Sitting on login screen, I've seen this happen once but wasn't able to break in the code to track down why I hit it so I am still investigating this one but it seems fairly low priority. Maybe we can move this to its own bug?
RE: Name, I added this here to make the emails that users get more personal. We do use it in cattle to in the name field but I could make the user enter it on the final page after you get you're reg email rather than before? But having personalized emails is kind nice IMO. Let me know you're thoughts.
